core-bundle/src/Security/Authentication/RememberMe/RememberMeTokenProvider.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Security\Authentication\RememberMe;
+
+use Contao\CoreBundle\Entity\RememberMe;
+use Contao\CoreBundle\Repository\RememberMeRepository;
+use Symfony\Component\Security\Core\Authentication\RememberMe\PersistentToken;
+use Symfony\Component\Security\Core\Authentication\RememberMe\PersistentTokenInterface;
+use Symfony\Component\Security\Core\Authentication\RememberMe\TokenProviderInterface;
+
+class RememberMeTokenProvider implements TokenProviderInterface
+{
+    public function __construct(private RememberMeRepository $repository)
+    {
+    }
+
+    public function loadTokenBySeries(string $series): PersistentToken
```

Comment:
<img width="737" alt="" src="https://user-images.githubusercontent.com/1192057/178517541-38239c7f-2f2a-40bb-bc41-2ebfc4d0c2c0.png">

Reply:
Same here? 🤷

Reply:
As discussed with @leofeyer we will keep it as it is and tackle/discuss the `@internal` usages all over Contao in a different PR.